ansible/ansible-content-parser
# Update ansible-lint to 6.22.2

Issue: No issues associated

## Description


Update ansible-lint embedded in ansible-content-parser to its latest version (6.22.2).

## Testing


Executed ansible-content-parser against  https://github.com/ansible-collections/amazon.aws.git and verified that the training data set (ftdata.jsonl) was successfully created.

### Steps to test

1. Build and install the updated ansible-content-parser from the PR (ref: https://github.com/ansible/ansible-content-parser/wiki/Build)
2. Execute
```
ansible-content-parser https://github.com/ansible-collections/amazon.aws.git /tmp/out
```
3. Make sure `/tmp/out` directory contains ftdata.jsonl file.

### Scenarios tested



See the Steps to test section above.
